docs/openlaw-object/README.md

```diff
@@ -0,0 +1,2141 @@
+
+Resume execution (load and run) of a compiled template when the initial execution by the [`execute` method](#execute) or [`executeForReview` method](#executeforreview) returns an object indicating that a template was missing (e.g., `{..., missingTemplate: true, missingTemplateName: "Employee Offer Letter", errorMessage: "the template Employee Offer Letter was not loaded"}`).
+
+```scala
+resumeExecution(
+  executionResult: TemplateExecutionResult,
+  jsTemplates: js.Dictionary[CompiledTemplate]
+): js.Dictionary[Any]
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `executionResult` | [`TemplateExecutionResult`](#templateexecutionresult) | **Required.** The nested object returned from the [`execute` method](#execute) and [`executeForReview` method](#executeforreview). |
+| `jsTemplates` | `Object` | **Required.** An object containing the compiled template to be executed that was missing from the initial execution. |
+
+Example
+
+```js
+// see examples above for #execute and #executeForReview for parameters
+const executionResult = Openlaw.execute(compiledTemplate.compiledTemplate, {}, params);
+const templatesForExecution = {[executionResult.missingTemplateName]: compiledTemplate.compiledTemplate};
+Openlaw.resumeExecution(executionResult.executionResult, templatesForExecution);
+```
+
+**Response**
+
+Returns an object containing a [`TemplateExecutionResult` object](#templateexecutionresult) (and key/value pairs indicating that no error has occurred and that no templates are missing) if execution is successful.
+
+Example
+
+```js
+{
+  executionResult: TemplateExecutionResult,
+  isError: false,
+  missingTemplate: false,
+  errorMessage: ""
+}
+```
+
+### getInitialParameters
+
+List the initial parameters with default values in an executed template returned from the [`execute` method](#execute) or [`executeForReview` method](#executeforreview).
+
+```scala
+getInitialParameters(
+  executionResult: TemplateExecutionResult
+): js.Array[js.Dictionary[String]]
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `executionResult` | [`TemplateExecutionResult`](#templateexecutionresult) | **Required.** The nested object returned from the [`execute` method](#execute) and [`executeForReview` method](#executeforreview). |
+
+Example
+
+```js
+// see examples above for #execute and #executeForReview for parameters
+const executionResult = Openlaw.execute(compiledTemplate.compiledTemplate, {}, params);
+Openlaw.getInitialParameters(executionResult.executionResult);
+```
+
+**Response**
+
+Returns an array of objects of initial parameters with default values.
+
+Example
+
+```js
+[
+  {
+    name: "Company Name",
+    value: "ABC, Inc."
+  },
+  {
+    name: "Number of Shares",
+    value: "1000"
+  }
+]
+```
+
+### validateContract
+
+Validate an executed template returned from the [`execute` method](#execute) or [`executeForReview` method](#executeforreview).
```

Comment:
Should we explain more what it means to validate the template / why validation is needed? Perhaps also something further regarding the transition from template to contract?

Reply:
@adridadou I have the same questions too. Do you have any suggestions on what we could add here?